src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDurationMapping.java

```diff
@@ -51,24 +53,19 @@ public static int getHearingDuration(SscsCaseData caseData, ReferenceDataService
             return duration;
         }
 
-        return DURATION_DEFAULT;
+        throw new ListingException("Missing hearing duration");
```

Comment:
This is not correct, this is not where the error is produced

Reply:
Removed, will ask with POs if they want a default value